### ISSUE TITLE
docker-compose fail 

### ISSUE THREAD

**Issue by tuian** [junelsolis/freeradius-admin#5]
### 0x00 Info
location:
https://github.com/junelsolis/freeradius-admin/blob/master/web/Dockerfile
```
FROM davidsarkany/laravel-on-php7-apache
RUN a2enmod rewrite && a2enmod ssl
```
davidsarkany/laravel-on-php7-apache
Page Not Found or need login.

So, `docker-compose build` will fail on `fradmin-web`

**Comment by junelsolis** [comment]
> FROM davidsarkany/laravel-on-php7-apache

This image no longer exists in the docker hub. Will work on a fix.

**Comment by junelsolis** [comment]
Fixed in commit c5ea20f2eabaad9d71f739ee9d6f1822737b284a. Thank you for reporting this issue.
